shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -34,18 +36,21 @@
   private static final String INDENT = "    ";
 
   private final MetaMasterClient mMetaMasterClient;
+  private final Map<String, MetricValue> mMetricsMap;
   private final PrintStream mPrintStream;
-  private String mInfoFormat = "%-25s %20s";
+  private String mInfoFormat = "%-30s %20s";
 
   /**
    * Creates a new instance of {@link MetricsCommand}.
    *
    * @param metaMasterClient client to connect to meta master client
    * @param printStream stream to print operation metrics information to
    */
-  public MetricsCommand(MetaMasterClient metaMasterClient, PrintStream printStream) {
+  public MetricsCommand(MetaMasterClient metaMasterClient, PrintStream printStream)
+      throws IOException {
     mMetaMasterClient = metaMasterClient;
     mPrintStream = printStream;
+    mMetricsMap = new TreeMap<>(mMetaMasterClient.getMetrics());
```

Comment:
not good to perform expensive/blocking operations in the constructor - could we call this from `run()` instead?

Reply:
Move back to run() method and add null check in the printMetric() method to avoid mvn findbugs complain.

```
  private void printMetric(String metricName, String nickName, boolean valueIsBytes) {
    if (mMetricsMap == null || !mMetricsMap.containsKey(metricName)) {
      return;
    }
```